src/carts/Dockerfile

```
@@ -38,5 +38,6 @@ WORKDIR /app
 
 ENV APPLICATION_NAME=${APPLICATION_NAME}
 ENV VERSION=${VERSION}
+ENV TNS_ADMIN=/usr/lib/oracle/19.3/client64/lib/network/admin/
```

Please put the Instant Client version as variable. 

```
##### Oracle Instant Client version
ARG clientVersion=19.3
...
```
```suggestion
ENV TNS_ADMIN=/usr/lib/oracle/${clientVersion}/client64/lib/network/admin/
```